config/routes.rb

```diff
@@ -95,7 +95,8 @@
       get "/badges/badge_achievements", to: redirect("/admin/badge_achievements")
       get "/badges/badge_achievements/award_badges", to: redirect("/admin/badge_achievements/award_badges")
 
-      # NOTE: @ridhwana These routes below will be deleted once we remove the admin_restructure feature flag, hence they've been regrouped them in this manner.
+      # NOTE: @ridhwana These routes below will be deleted once we remove the
```

Comment:
This change is completely unrelated to this PR, but I had a Rubocop error when merging back `master` to resolve a conflict, so I fixed it.